llvm/lib/CodeGen/GlobalISel/Utils.cpp

```diff
@@ -1618,13 +1618,6 @@ int64_t llvm::getICmpTrueVal(const TargetLowering &TLI, bool IsVector,
   llvm_unreachable("Invalid boolean contents");
 }
 
-bool llvm::shouldOptForSize(const MachineBasicBlock &MBB,
-                            ProfileSummaryInfo *PSI, BlockFrequencyInfo *BFI) {
-  const auto &F = MBB.getParent()->getFunction();
-  return F.hasOptSize() || F.hasMinSize() ||
```

Comment:
This lost the hasMinSize check

Reply:
`F.hasOptSize()` (now called in `shouldOptimizeForSize()`) already calls `hasMinSize()`.
https://github.com/llvm/llvm-project/blob/927af63fddb8e34f23b2974f812156767988ec5f/llvm/include/llvm/IR/Function.h#L707-L709